README.md

```diff
@@ -248,26 +248,35 @@ See [education.md](docs/education.md)
 ## Start your own Model
 
 When starting `python -m causing.examples example` after cloning / downloading the Causing repository you will find
-the example results described above in the _output_ folder. They are given as PDF files and
-the single graphs are also provided as PNG files for further use.
+the example results described above in the _output_ folder. The results are saved in SVG files.
 
 See `causing/examples` for the code generating these examples.
 
 To run a model, you have to provide the following information, as done in the example code below:
 
 - Define all your model variables as SymPy symbols.
-- In _define_equations_ define a python SymPy function containing the model
-equations and returning them in topological order, that is, in order of computation.
+- Provide the model equations in topological order, that is, in order of computation.
 Note that in Sympy some operators are special, e.g. Max() instead of max().
-- In _model_dat_, the dictionary to be returned, further specify
-    - _xvars_: exogenous variables corresponding to data _xdat_
+- Then the model is specified with:
+    - _xvars_: exogenous variables in desired order
```

Comment:
remove "in desired order": there is no required order for xvars, we just need an ordering for the yvars